charts/posit-chronicle/values.yaml

```diff
@@ -0,0 +1,80 @@
+image:
+  repository: "ghcr.io/rstudio/chronicle"
+  tag: "latest"
```

Comment:
Similar comment about using the latest tag. Our other charts use the appVersion, could that make sense here too?

Reply:
Can do! We can use [v2023.11.2](https://github.com/rstudio/chronicle/pkgs/container/chronicle) which we're publishing now.
That's a good callout, we were just talking on our team about which image to use